python/tests/expression/test_literals.py

```diff
@@ -15,24 +15,18 @@
 # specific language governing permissions and limitations
 # under the License.
 
-import math
 import uuid
-from datetime import datetime
 from decimal import Decimal
 
-import dateutil.parser
 import pytest
-from fastavro.write import LOGICAL_WRITERS
```

Comment:
nit: if we do not need this, we can probably remove it from tox test dependency as well

Reply:
Agreed! I also think these tests that were using this could be expanded on a bit to be more comprehensive, maybe by using a pytest.parametrize of an input set. Maybe that's something that can be done in a follow-up PR.